internal/applier/bundledeployment_applied.go

```diff
@@ -1,56 +1,36 @@
 package applier
```

Comment:
Unrelated to this specific PR, but should this file instead be called `bundledeployment_applier.go`? It is `bundledeployment_applied.go` right now.

Reply:
Whoops. Done in https://github.com/openshift/platform-operators/pull/41/commits/50ba86b0bdada6d75eb6be29359d7f0baebdff98 to so the overall diff is still preserved in the first commit.